scraper/data/reinsaatRawData.csv

```diff
@@ -0,0 +1,718 @@
+"name","category","subcategory","url","Sow/ plant outdoors","Article number","Portion contents","Thousand grain weight","Plant outdoors","Harvest","Pre-cultivation","Distances","Blossoming period","Cultivation","Spacing","Required quantity of seeds","Sowing","Distances ","Sowing depth","Sowing depth ","Planting","Distance","Propagator","Sowing ","Thousand seeds mass","Propagator ","Planting  ","Required quantity","Sowing outdoors","Sowing depth ","Distances ","Thousand seeds mass ","Distances approx. ","Harvest ","1stharvest","Germination temperature ","Germination temperature","Sowing ","1st harvest","Prpagator","Sow/ plant in greenhouse","Pre-cultivation ","Planting   ","Direct sowing","Required quantity of seeds ","Development time approx. ","Sowing/Pre-culttivation ","Thousand grain mass","Precision seeds","Thousand seeds mass","Sowing              ","Sowing direct outside ","Pre-cultivation ","Planting           ","Harvesting    ","Harvesting ","Ripening period","Required quantity of seeds ","Planting ","Distance        Ko49:","                       Ko51","              to a scale of 10 0 (mild) – 10 (extremely hot)."
```

Comment:
We don't need the CSVs within the repo, we only store them in nextcloud and import from there. See #248.